test/viewModels/ArrivalStatusViewModelSpec.scala

```diff
@@ -80,7 +80,7 @@ class ArrivalStatusViewModelSpec extends SpecBase with Generators with ScalaChec
               "viewArrivalNotifications.table.action.viewErrors"
             )
 
-            ArrivalStatusViewModel(updatedArrival)(frontendAppConfig).status mustBe "movement.status.ArrivalSubmittedNegativeAcknowledgement"
+            ArrivalStatusViewModel(updatedArrival)(frontendAppConfig).status mustBe "movement.status.p4.ArrivalSubmittedNegativeAcknowledgement"
```

Comment:
The message key looks to have changed to make this camel case but test doesn't reflect it?